src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -129,6 +136,95 @@ public void workspaceAuthz(AuthenticatedUserRequest userReq, UUID workspaceId, S
               userReq.getEmail(), action, workspaceId.toString()));
   }
 
+  /**
+   * Wrapper around Sam client to grant a role to the provided user.
+   *
+   * <p>This operation is only available to MC_TERRA stage workspaces, as Rawls manages permissions
```

Comment:
What is an MC_TERRA stage workspace?
I think you mean an MC_WORKSPACE?

Reply:
Nice catch, thanks!